pkg/authn/authn.go

```diff
@@ -0,0 +1,31 @@
+package authn
```

Comment:
We could change this to `authclaims` to avoid clashing in other places. The content also relates more to claims than generic authn concepts